app/src/main/java/com/jetbrains/handson/mpp/mobile/MainActivity.kt

```diff
@@ -28,18 +27,28 @@ class MainActivity : AppCompatActivity(), ApplicationContract.View {
     }
 
     override fun showResults(result: List<JourneyTableDataElem>) {
-        findViewById<TextView>(R.id.search_results).text = result.toString()
+        findViewById<RecyclerView>(R.id.search_results).adapter = SearchResultAdapter(result)//.text = result.toString()
```

Comment:
I think I commented on this elsewhere too, perhaps some rebasing needs to be done to stop these changes showing up twice